libs/parallelism/executors/include/hpx/executors/parallel_executor.hpp

```diff
@@ -165,6 +166,23 @@ namespace hpx { namespace execution {
         {
         }
 
+        friend parallel_policy_executor tag_invoke(
+            hpx::execution::experimental::make_with_hint_t,
+            parallel_policy_executor const& exec,
+            hpx::threads::thread_schedule_hint hint)
+        {
+            auto exec_with_hint = exec;
+            exec_with_hint.schedulehint_ = hint;
+            return exec_with_hint;
+        }
+
+        friend hpx::threads::thread_schedule_hint tag_invoke(
+            hpx::execution::experimental::get_hint_t,
+            parallel_policy_executor const& exec)
+        {
+            return exec.schedulehint_;
+        }
+
```

Comment:
Should we add overloads for arbitrary executors. Those obviously shouldn't do anything. It would make things generically work for all executors, however.

Reply:
Isn't that what `prefer` is for? I agree we should expand the support, but not with overloads that don't do anything.